### ISSUE TITLE
Can't get Fogbugz to show up under issue tracker

### ISSUE THREAD

**Issue by tinystarship** [jozefizso/teamcity-fogbugz#3]
I installed your plugin today. Fogbugz  (2017.2.4) recognizes the plugin, but doesn't display the version. This is what it shows:

teamcity-fogbugzIntegrates FogBugz issue tracker with TeamCity. | @Version@ | @VendorName@ | <TeamCity Data Directory>/plugins/teamcity-fogbugz-master.zip

Any ideas on what I am doing wrong?

**Comment by jozefizso** [comment]
Where does the `teamcity-fogbugz-master.zip` file comes from? The official release is named [teamcity-fogbugz.zip](https://github.com/jozefizso/teamcity-fogbugz/releases/download/v1.0.0/teamcity-fogbugz.zip)

**Comment by tinystarship** [comment]
Yes, I am an idiot. I completely missed the Release section and tried to upload the actual source code as a .zip, it allowed me to which is why I had an issue. 

thank you for your extremely quick reply. I am working again!
